tutorials/language/tedd.py

```diff
@@ -63,102 +63,102 @@
-# represents a  
-# stage.  Stage name shows in the top row and compute shows in the bottom row.   
-# The middle rows are for IterVars, the higher the outer, the lower the inner. 
-# An IterVar row contains its index, name, type, and other optional information. 
-# Let's use the W.shared stage as an example.  The top row tells  
-# its name, "W.shared", and memory scope, "Shared".  Its compute is  
-# :code:`W(ax0, ax1, ax2, ax3)`. 
-# Its outer most loop IterVar is ax0.ax1.fused.ax2.fused.ax3.fused.outer,  
-# indexed with 0, of kDataPar, bound to threadIdx.y, and with range(min=0, ext=8).  
-# You can also tell 
-# IterVar type with the index box color, shown in the legend. 
-# 
-# If a stage doesn't compute_at any other stage, it has an edge directly to the  
-# ROOT node.  Otherwise, it has an edge pointing to the IterVar it attaches to,  
-# such as W.shared attaches to rx.outer in the middle compute stage. 
+# Now, let us take a close look at the second schedule tree.  Every block under ROOT
+# represents a
+# stage.  Stage name shows in the top row and compute shows in the bottom row.
+# The middle rows are for IterVars, the higher the outer, the lower the inner.
+# An IterVar row contains its index, name, type, and other optional information.
+# Let's use the W.shared stage as an example.  The top row tells
+# its name, "W.shared", and memory scope, "Shared".  Its compute is
+# :code:`W(ax0, ax1, ax2, ax3)`.
+# Its outer most loop IterVar is ax0.ax1.fused.ax2.fused.ax3.fused.outer,
+# indexed with 0, of kDataPar, bound to threadIdx.y, and with range(min=0, ext=8).
+# You can also tell
+# IterVar type with the index box color, shown in the legend.
+#
+# If a stage doesn't compute_at any other stage, it has an edge directly to the
+# ROOT node.  Otherwise, it has an edge pointing to the IterVar it attaches to,
+# such as W.shared attaches to rx.outer in the middle compute stage.
 #
 
-###################################################################### 
-# .. note:: 
-# 
-#   By definition, IterVars are internal nodes and computes are leaf nodes in 
-#   a schedule tree.   The edges among IterVars and compute within one stage are 
+######################################################################
+# .. note::
+#
+#   By definition, IterVars are internal nodes and computes are leaf nodes in
+#   a schedule tree.   The edges among IterVars and compute within one stage are
 #   omitted, making every stage a block, for better readability.
 #
 
-tedd.viz_itervar_relationship_graph(s, dot_file_path = '/tmp/itervar.dot') 
-#tedd.viz_itervar_relationship_graph(s, show_svg = True) 
+tedd.viz_itervar_relationship_graph(s, dot_file_path = '/tmp/itervar.dot')
+#tedd.viz_itervar_relationship_graph(s, show_svg = True)
 
 ######################################################################
 # .. image:: https://github.com/dmlc/web-data/raw/master/tvm/tutorial/tedd_itervar_rel.png
 #      :align: center
 #      :scale: 100%
```

Comment:
Remove this line. `:scale` will be ignored because Sphinx cannot get the image size.